portfolio/src/main/webapp/index.html

```diff
@@ -8,10 +8,11 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Erica Smith's Portfolio</h1>
+      <p>I am a rising sophomore at NC A&T SU</p>
+      <img src="/images/aggiesdo.png" alt="AggiesDo">
```

Comment:
I don't see this file in the commit. Did you `git add` it?

Reply:
Do you mean the image?

Reply:
Yep, looks like you added it, thanks!